starters/next-drupal-starter/pages/index.jsx

```diff
@@ -90,6 +89,12 @@ export async function getServerSideProps(context) {
       res: context.res,
     });
 
+    const sortedArticles = sortChar({
+      data: articles,
+      key: "title",
+      direction: "asc",
+    });
+
     if (!articles) {
       throw new Error(
         "No articles returned. Make sure the objectName and params are valid!"
```

Comment:
I would expect the `if (!articles)` check to happen before passing articles in to `sortChar` since `sortChar` will likely throw an error if articles is undefined.